src/panels/ClusterPropertiesPanel.ts

```diff
@@ -64,6 +65,53 @@ export class ClusterPropertiesDataProvider implements PanelDataProvider<"cluster
         }
     }
 
+    private async _handleAbortClusterOperation(webview: MessageSink<ToWebViewMsgDef>) {
+        try {
+            const poller = await this.client.managedClusters.beginAbortLatestOperation(this.resourceGroup, this.clusterName);
+
+            poller.onProgress(state => {
+                if (state.status === "failed") {
+                    const errorMessage = state.error ? getErrorMessage(state.error) : "Unknown error";
+                    webview.postErrorNotification(errorMessage);
+                }
+            });
+
+            // Update the cluster properties now the operation has started.
+            await this._readAndPostClusterProperties(webview);
+
+            // Wait until operation completes.
+            await poller.pollUntilDone();
+        } catch (ex) {
+            const errorMessage = getErrorMessage(ex);
+            webview.postErrorNotification(errorMessage);
+        }
+    }
+
+    private async _handleReconcileClusterOperation(webview: MessageSink<ToWebViewMsgDef>) {
+        try {
+            const getClusterInfo = await this.client.managedClusters.get(this.resourceGroup, this.clusterName);
+            const poller = await this.client.managedClusters.beginCreateOrUpdate(this.resourceGroup, this.clusterName, {
+                location: getClusterInfo.location
+            });
+
+            poller.onProgress((state) => {
```

Comment:
I think we can have an extra progress check for "cancelled" here (as we do for start/stop).

The reason we skipped that for Abort was because Abort effectively _is_ a cancel operation, so I don't think it makes sense to cancel it. But Reconcile is an update, which I think _could_ be cancelled?

I guess we could test this by kicking off a long-running reconcile and trying to abort it from the CLI.